packages/data-connect/test/queries.test.ts

```diff
@@ -29,86 +29,87 @@ import {
   queryRef,
   QueryResult,
   SerializedRef,
-  subscribe,
-  terminate,
   SOURCE_CACHE,
-  SOURCE_SERVER
+  SOURCE_SERVER,
+  subscribe,
+  terminate
 } from '../src';
 
-import { setupQueries } from './emulatorSeeder';
+// import { setupQueries } from './emulatorSeeder';
 import { getConnectionConfig, initDatabase, PROJECT_ID } from './util';
 
 use(chaiAsPromised);
 
-interface Task {
+interface Post {
   id: string;
-  content: string;
+  description: string;
 }
-interface TaskListResponse {
-  posts: Task[];
+interface PostListResponse {
+  posts: Post[];
 }
 
 const SEEDED_DATA = [
   {
     id: crypto.randomUUID(),
-    content: 'task 1'
+    description: 'task 1'
   },
   {
     id: crypto.randomUUID(),
-    content: 'task 2'
+    description: 'task 2'
   }
 ];
 const REAL_DATA = SEEDED_DATA.map(obj => ({
   ...obj,
   id: obj.id.replace(/-/g, '')
 }));
-function seedDatabase(instance: DataConnect): Promise<void> {
-  // call mutation query that adds SEEDED_DATA to database
-  return new Promise((resolve, reject) => {
-    async function run(): Promise<void> {
-      let idx = 0;
-      while (idx < SEEDED_DATA.length) {
-        const data = SEEDED_DATA[idx];
-        const ref = mutationRef(instance, 'seedDatabase', data);
-        await executeMutation(ref);
-        idx++;
-      }
-    }
-    run().then(resolve, reject);
-  });
-}
+
 async function deleteDatabase(instance: DataConnect): Promise<void> {
```

Comment:
It looks like you're trying to get the database into a "clean state" after each run. Note that these tests often run in parallel so make sure that if that were to happen that two such independent test runs would not interfere with each other. Alternately, consider writing the tests such that data does not need to be cleaned up (that's what I did for Kotlin). Feel free to address this in a follow-up PR, if desired.

Reply:
So in CI, the test runs in prod, so there's no way that I can run these tests without having to clean-up afterwards. So what I do here is in the `beforeEach`, I create a `testId` that ends up marking the posts created for a specific test, and therefore not conflicting with any other test

Reply:
Discussed offline. The concern was around unbounded database growth rather than tests not supporting being run in parallel. I am satisfied.